includes/admin/views/kco-support.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * Klarna support tab in the settings page.
+ *
+ * @package  Klarna_Checkout/Includes/Admin/Views
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( isset( $_POST['submit'] ) ) {
+
+	$send = true;
+	if ( ! is_email( $_POST['email'] ) ) {
+		$send = false;
+	}
+
+	$from    = sanitize_email( $_POST['email'] );
+	$to      = 'support@krokedil.se';
+	$subject = sanitize_text_field( $_POST['subject'] );
+	$message = sanitize_textarea_field( $_POST['description'] );
+	$headers = array(
+		'From: ' . $from,
+		'Reply-To: ' . $from,
+		'Content-Type: text/html; charset=UTF-8',
+	);
+
+
+	$attachment = array(
+		WC_LOG_DIR . sanitize_text_field( $_POST['klarna-checkout'] ),
+		WC_LOG_DIR . sanitize_text_field( $_POST['klarna-order-management'] ),
+		WC_LOG_DIR . sanitize_text_field( $_POST['fatal-error-log'] ),
+		WC_LOG_DIR . sanitize_text_field( $_POST['additional-log'] ),
+	);
+
+	foreach ( $_POST as $key => $value ) {
+		if ( strpos( $key, 'additional-log-' ) !== false ) {
+			$attachment[] = WC_LOG_DIR . sanitize_text_field( $value );
+		}
+	}
+
+	$attachment = array_filter(
+		$attachment,
+		function( $path ) {
+			return file_exists( $path ) && is_readable( $path );
+		},
+	);
+
+	if ( 'on' === $_POST['system-report-include'] ) {
+		$system_report = sanitize_textarea_field( $_POST['system-report'] );
+		$filename      = 'system-report-' . date( 'Y-m-d' ) . '.txt';
+		$filepath      = WP_CONTENT_DIR . '/uploads/' . $filename;
+
+		if ( file_put_contents( $filepath, $system_report ) ) {
+			$attachment[] = $filepath;
+		}
+	}
+
+	if ( $send ) {
+		// wp_mail( $to, $subject, $message, $headers, $attachment );
+	}
+}
+
+function kco_log_wrapper( $title, $log ) {
+	// Normalized name intended to be form and CSS friendly.
+	$name = esc_attr( str_replace( ' ', '-', strtolower( $title ) ) );
+	?>
+	<div class="log-wrapper <?php esc_attr_e( $name ); ?>">
+		<h3><?php echo $title; ?></h3>
+		<?php
+		if ( empty( $log ) ) {
+			echo '<p class="' . $name . '">No log available.</p>';
+		}
+		?>
+		<select class="kco-log-option wc-enhanced-select <?php echo $name; ?> " name="<?php echo $name; ?>">
+			<?php foreach ( $log as $date => $path ) : ?>
+				<option name="<?php echo $name; ?>" value="<?php esc_attr_e( $path ); ?>"><?php echo esc_html( "{$path} ({$date})" ); ?></option>
+			<?php endforeach; ?>
+		</select>
+		<a class="view-log">View log</a>
+		<textarea class="log-content" readonly></textarea>
+	</div>
+	<?php
+}
```

Comment:
This should also be moved to a separate file. The view should not contain any logic